src/Cake.Curl/ArgumentsExtensions.cs

```diff
@@ -77,6 +78,20 @@ internal static class ArgumentsExtensions
             {
                 arguments.Append("--retry-connrefused");
             }
+
+            if (settings.MaxTime.HasValue)
+            {
+                arguments.AppendSwitch(
+                    "--max-time",
+                    settings.MaxTime.Value.TotalSeconds.ToString(CultureInfo.CurrentCulture));
```

Comment:
Here, we can simply say `settings.MaxTime.Value.TotalSeconds` since the compiler will add the call to `ToString()` when using [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated). Also, when invoking `ToString()` on a `Double` the usage of the current culture is implicit.

Reply:
@ecampidoglio There is no string interpolation used here - but it can be used. I can rewrite the code to `${settings.MaxTime.Value.TotalSeconds}` which does the same as current code. Your suggestion does not work because there is no implicit conversion from double to string.

Reply:
@ecampidoglio Correct — it seems I was a little too quick commenting here. 😳
I don't see the value in using string interpolation if there are no string literals involved. It would be enough to simply say `settings.MaxTime.Value.TotalSeconds.ToString()`.

Reply:
@ecampidoglio If I use ToString() on double variable,  I got warning that I should specify culture setting. So it seems that there are two options only - use string interpolation or use current code.

Reply:
ReSharper is issuing [that warning](https://www.jetbrains.com/help/resharper/SpecifyACultureInStringConversionExplicitly.html). I tend to evaluate warnings that come from tools in their specific context before deciding whether I want to act on them or not.

In this case, the format of the decimal value passed to the `--max-time` option _has to_ conform to the system's culture or curl will give you an error (see curl/curl#2426); I think this detail is important enough that it should be explicit in the code.

So yes, let's keep `settings.MaxTime.Value.TotalSeconds.ToString(CultureInfo.CurrentCulture)`. 👍